### DISCUSSION TITLE
Auto playing data stories

### DISCUSSION THREAD

**Discussion by arky** [TerriaJS/terriajs#5106] [Q&A] [answered]
When you share a link with a data story. The user has to select Yes in the 'This map contains a story' dialog to view the story. Is there an option to automatically play the data story without user interaction? The use case for this when you embedded maps into other pages via an iframe or share the link to social media streams. 

Testcase

https://nsw.digitaltwin.terria.io/#share=s-8BfcLqkdgweUrqTNTJUusfLs2VX

![Screenshot from 2020-12-20 04-22-48](https://user-images.githubusercontent.com/808895/102699894-17a48280-427b-11eb-8dd4-27e590fa99f6.png)

**Comment by arky** [comment, marked as answer]
The documentation refers to option 'playStory=1' to enable this feature. 

https://nsw.digitaltwin.terria.io/#share=s-8BfcLqkdgweUrqTNTJUusfLs2VX&playStory=1

**Comment by christopherknutson** [comment]
This posted helped a lot!  Is there a way to have stories not only autoplay, but also _automatically rotate through scenes on a timer_ (eg. every 10 seconds)?  Thanks!

**Comment by arky** [comment]
@christopherknutson Glad that helped you. AFAIK scene transition is triggered by user click or swipe. I don't think there is a feature to loop through the scenes on set time. Do file a issue for that feature.

**Reply by christopherknutson** [reply]
Thanks @arky!  Just filed an issue: https://github.com/TerriaJS/terriajs/issues/6026
